test/views/html/incomeTaxComparison/MainViewSpec.scala

```diff
@@ -110,7 +110,6 @@ class MainViewSpec extends TaiViewSpec with ViewModelHelper {
       doc must haveLinkWithUrlWithID(
         "returnToChooseTaxYearLink",
         controllers.routes.WhatDoYouWantToDoController.whatDoYouWantToDoPage().url)
-      doc must haveLinkWithText(messages("your.paye.income.tax.overview"))
```

Comment:
How come this has been removed? Is it because it is not part of the header but part of the main template now?

Reply:
I will create another method as this one won't work because it searches the link in a different tag

Reply:
I've put this back, the message was wrong in the MainView. Well spotted!